module-5/M5/config/db.js

```diff
@@ -0,0 +1,5 @@
+import { Sequelize } from 'sequelize';
+
+const sequelize = new Sequelize('postgres://db_user:135256@localhost:5432/portfolio' , {dialect: 'postgres', logging: false});
```

Comment:
Looks like we have one more area we need to use Env variables for settings instead of hardcoded passwords.